app/src/main/res/layout/fragment_discover.xml

```diff
@@ -34,16 +34,17 @@
             android:background="@color/purpleStandard"
             android:elevation="4dp"/>
 
-        <TextView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:text="@{viewModel.tempOutput}"
-            android:elevation="10dp"
-            android:textColor="@color/textStandard"
-            android:layout_margin="@dimen/default_margin"
-            app:layout_constraintEnd_toEndOf="parent"
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/animal_list_recycler_view"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:adapter="@{viewModel.adapter}"
+            app:gridLayoutManager="@{2}"
```

Comment:
Why not put this in the adapter as a static field, will be nice for when the user can configure

Reply:
I meant ViewModel, doh